applications/ShapeOptimizationApplication/test_examples/05_External_Fluid_Adjoint_Transient_Analyzer/base_case/impose_parabolic_inlet_process.py

```diff
@@ -0,0 +1,31 @@
+import KratosMultiphysics
+from math import cos
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ImposeParabolicInletProcess(Model, settings["Parameters"])
+
+class ImposeParabolicInletProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        KratosMultiphysics.Process.__init__(self)
+        
+        self.model_part = Model[settings["model_part_name"].GetString()]
+
+    def ExecuteInitialize(self):
+        for node in self.model_part.Nodes:
```

Comment:
if you decide to keep this then please use the function form `kratos/utilities/variable_utils` for this